components/record/MediaPresentation.vue

```diff
@@ -134,7 +121,14 @@
           this.oEmbedData = { error: err };
         });
       }
+    },
+
+    methods: {
+      imageLink(pdf) {
+        return pdf ? this.media.about : this.$options.filters.proxyMedia(this.media.about, this.europeanaIdentifier) + '&disposition=inline';
```

Comment:
Update the proxyMedia filter to accept arbitrary additional parameters as an optional third param, like:
```js
Vue.filter('proxyMedia', (mediaUrl, europeanaId, params = {}) => {
```

Then pass disposition to the filter in this call.